applying-stpa/l1_reqs.txt

```diff
@@ -0,0 +1,228 @@
+REQ-135: OpenAPS glucose level projections must be verified, and projection calculations must be refined if projections are regularly inaccurate
+REQ-136: OpenAPS should not suspend insulin dosing when the human is not close to experiencing hypoglycemia
+REQ-137: OpenAPS should inform the human actor when it suspends the insulin pump (due to a problem with the insulin pump)
+REQ-138: OpenAPS shall employ a monitoring task to ensure that all safety-related threads in the application are triggered by the kernel on a timely basis (e.g. watchdog timer)
+REQ-139: OpenAPS should notify user when other components' clock time differs from RasPi system clock
+REQ-140: OpenAPS must suspend insulin dosing within X minutes after the human is experiencing or close to experiencing hypoglycemia (same as controller constraint)
+REQ-141: OpenAPS rig setup instructions should be clear, complete, easy to find and cover all possible hardware
+REQ-142: There should be a way for errors in the OpenAPS instructions to be reported
+REQ-143: There should only be one set of official OpenAPS instructions online, which contains all the information that needs to be known to use OpenAPS safely
+REQ-144: OpenAPS rig hardware setup should be tested for robustness and fragiliity
+REQ-145: OpenAPS should send a warning to human actor when human actor hasn't set up the rig matching with the configurations
+REQ-146: Whenever a change is made to the OpenAPS system or the compatible versions change, the docs should be updated
+REQ-147: There should only be one set of official OpenAPS documentation available for the human actor to read, which contain all information that is in any other instructions found online
+REQ-148: If there is a change to the system, the documentation should be updated to reflect this
+REQ-149: OpenAPS docs regarding other components should be reviewed regularly, in keeping with the latest recommended approach
+REQ-150: OpenAPS should notify user when SD card is of the wrong size or format
+REQ-151: Important information in docs should be easy to find in the hardware setup area of the docs
+REQ-152: The OpenAPS docs must be easy to navigate, with a search bar
+REQ-153: OpenAPS should detect if incompatible insulin pump software is being used and indicate this to the user
+REQ-154: When new updates have been tested to be compatible, OpenAPS should issue an email notification
+REQ-155: OpenAPS should list compatible versions in the documentation
+REQ-156: OpenAPS should check that human is using compatible versions of components. New versions of components should be tested
+REQ-157: Add configuration check between systems including of their software/firmware versions used and only allow system to work with insulin pumps that have been properly tested
+REQ-158: Add prompts in OpenAPS UI that insists that human reviews and understands instructions prior to/during/after setting up the rig.
+REQ-159: Nice to have is an educational video on how to set up the rig
```

Comment:
Probably this should grouped under some sort of training -   Open question for nicole - is should requirements contain "Nice to have"?